modules/instruments/test/TestPlan.md

```diff
@@ -0,0 +1,18 @@
+# Instrument Test Plan
+
+1. Verify that in order to access the module, the user must either have the `Across all sites access
+candidate profiles` permission, or have the 'Edit Candidate Parameters' permission and be at the same
```

Comment:
i don't believe adit candidate parameters is necessary. that seems like the necessary permission to access timepoint_list. the instrument is reached from instrument_list module.

Reply:
I can remove permissions verification as there if you wish but without, a user can't reach this module, that is why I included them. Permission are effectively covered in instrument_list module.

'All candidate' is required if the candidate is not at the same center as the user.

Reply:
@PapillonMcGill My hesitation is due to the fact that noone will update this testplan if the permission requirements of timepoint_list and or instrument_list change... so how about we just refer to these modules instead of repeating it here ?

i would say something along the lines of 
```
Verify that in order to access the module the user has meets one of the following criteria:
 - User can access the candidate_list module (Access Profile) then click on a PSCID to get to the timepoint_list module then click on a timepoint to get to the instrument_list module and finally click on an instrument to access the instrument itself. 
   _**Note:** All steps before clicking on the instrument itself should be handled by their own modules and necessary permissions to access these modules should be part of their respective testplans._
 - User is provided a direct link to the instrument.
```

Reply:
@ridz1208 Ok, will modify.

Reply:
currently if you have a direct link to an instrument you dont need any permissions, however I might change that in a PR to come

Reply:
@PapillonMcGill I linked the PR so the testplan should add the data_entry requirement to the text i suggested

Reply:
@ridz1208  If you change a functionality, you should also change the documentation in the same PR.

Reply:
lol which ever PR makes it in first, we'll fix the other one i guess